### DISCUSSION TITLE
Raise Governor Limit on Base

### DISCUSSION THREAD

**Discussion by ArmandKhatri** [wormhole-foundation/wormhole#3745] [Governance] [answered]
The governor limit on Wormhole for Base is currently set to $500k, which is 90% less than other EVM networks.

With the launch of the Wormhole native ETH bridge, we want to avoid a situation where transactions are rate limited as a result of low caps. To ensure a smooth user experience and continued growth for Wormhole within the Base ecosystem, can the caps be raised to meet parity of other EVM networks.

Current 24hr Limit: $500,000
Proposed 24hr Limit: $5,000,000

**Comment by claudijd** [comment]
I pulled the public data on the Base chain for Guardian consideration:

**Base**

Base 1 month numbers:
Wormhole chain id 30 token bridge transfers over the past 31 days between sequences 13372 and 32722:
Notional usd average : $761,246.60
Notional usd total   : $23,598,644.63

Base 6 month numbers:
Wormhole chain id 30 token bridge transfers over the past 176 days between sequences 8 and 32551:
Notional usd average : $227,235.65
Notional usd total   : $39,993,474.73

It's clear that the current limit is below the current usage, and is 1.5x over the 1 month daily average, which appears to track similarly what Polygon is also seeing as a result of better EVM bridging experience.  Looking at the last 24hr sliding window, Base's current governor limit 78.67% of the utilization.  Based on the usage criteria, the proposed $5M would set the limit at 6.6x the current utilization and if the Guardians chose something more conservative at say $1M or $2M you'd have 1.3x and 2.6x respectively.

With the above data provided, I encourage the Guardians to weigh on where they believe the limit should be adjusted to.  I'm happy to send the PR once Guardians have weighed in.

**Comment by 1xCL** [comment]
I think $2m would suffice the current usage. We can change the limit higher after if data shows it's recommended.

**Comment by shieldstar** [comment, marked as answer]
I agree that $2m would be a good start here.

**Reply by jonastra** [reply]
I agree with my colleague, it makes perfect sense to increase the limit above the current usage, at the same time reducing risk by not increasing way past the current limit.

**Reply by claudijd** [reply]
PR is up: https://github.com/wormhole-foundation/wormhole/pull/3756

**Comment by Yannimotoo** [comment]
No objections to this proposal from our side. $2m limit for now seems reasonable and we can always increase it if necessary.
